src/ng/compile.js

```diff
@@ -3370,7 +3531,7 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
       if (EVENT_HANDLER_ATTR_REGEXP.test(name)) {
         throw $compileMinErr('nodomevents',
             'Interpolations for HTML DOM event attributes are disallowed.  Please use the ' +
-                'ng- versions (such as ng-click instead of onclick) instead.');
+                'ng- versions (such as ng-click or ng-on-click rather than onclick) instead.');
```

Comment:
Same here.

Reply:
These are definitely long. I guess if I'm changing them already then there's no harm also making them shorter? Think we should do that in this PR?

Reply:
If you feel you have capacity to do that. (And the saved bytes would make up for the extra sanitizer code that we have landed here :-) )

Reply:
I think I can do it along with the other changes 👍 

I'll adjust the docs to ensure everything in this message is in there, and I think this message can be cut down to just the first sentence. Sound ok?

Reply:
Done: a3b1f41f0350880a4b9e7eefaa13e67d0bf45776